mmv1/third_party/terraform/services/compute/resource_compute_network_test.go.erb

```diff
@@ -113,6 +113,121 @@ func TestAccComputeNetwork_routingModeAndUpdate(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
+func TestAccComputeNetwork_bgpBestPathSelectionModeAndUpdate(t *testing.T) {
+   t.Parallel()
+
+
+   var network compute.Network
+   suffixName := acctest.RandString(t, 10)
+   networkName := fmt.Sprintf("tf-best-bgp-path-selection-mode-%s", suffixName)
+
+
+   acctest.VcrTest(t, resource.TestCase{
+       PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+       ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+       CheckDestroy:             testAccCheckComputeNetworkDestroyProducer(t),
+       Steps: []resource.TestStep{
+           {
+               Config: testAccComputeNetwork_best_bgp_path_selection_mode(networkName, "LEGACY"),
+               Check: resource.ComposeTestCheckFunc(
```

Comment:
We could use the existing check helper functions as suggested below.

Check: resource.ComposeTestCheckFunc(
					resource.TestCheckResourceAttr("google_compute_network. acc_network_bgp_best_path_selection_mode", "bgp_best_path_selection_mode", "LEGACY"),
				),

Reply:
updated